install/homebrew.md

```diff
@@ -9,12 +9,19 @@ This installation method is designed for easy install and upgrades, and uses san
 
 ##### Prerequisites
 
-In order to install and Airsonic on macOS all you need is [Homebrew](https://brew.sh).
+In order to install and Airsonic on macOS you need Java 8 runtime and [Homebrew](https://brew.sh).
 
-Airsonic requires a recent Java runtime. The formula requires this cask, but it will not be installed automatically.
+Download the _JDK 8 .dmg_ package from the [JDK download page](https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) ([direct download link](http://download.oracle.com/otn-pub/java/jdk/8u191-b12/2787e4a523244c269598db4e85c51e0c/jdk-8u191-macosx-x64.dmg)).
```

Comment:
Could your please remove the direct link, we should not link to something that will surely be updated to avoid installing unsecure versions.

Reply:
You bet